0000-template-rfc.md

```diff
@@ -0,0 +1,57 @@
+- Feature Name: (fill me in with a unique ident, my_awesome_feature)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty)
+- IPFS Issue: (leave this empty)
```

Comment:
I would add a field here for `Prior discussions` that would allow to add bigger context and possible previous commentary that lead to establishment of the RFC as it is seen in here.

Reply:
The headers section is getting ugly and overloaded. Instead I tweaked the "Rationale" section to also call for links to prior discussions.